test/helpers/generateSsoProviders.js

```diff
@@ -7,15 +7,14 @@ describe('generateSsoProviders', () => {
   const returnPath = '/return-url';
   const escapedReturnUrl = escape([envVars.baseUrl, returnPath].join(''));
 
+  const ssoProvidersFromEnv = envVars.ssoProviderText && envVars.ssoProviderUrl
```

Comment:
@pgoetze I added this so tests won't fail when you haven't defined any providers in `ENV`.